test/helper/IncidentP5HelperSpec.scala

```diff
@@ -124,15 +124,21 @@ class IncidentP5HelperSpec extends SpecBase with ScalaCheckPropertyChecks with G
       }
 
       "coordinatesRow" - {
-        "must return a row" in {
-
+        "must return a row if gnss exists" in {
           val helper = new IncidentP5Helper(incidentType03, refDataService)
           val result = helper.coordinatesRow.value
 
           result.key.value mustBe "Coordinates"
-          result.value.value mustBe "coordinates"
+          result.value.value mustBe s"(${incidentType03.Location.GNSS.get.latitude}, ${incidentType03.Location.GNSS.get.longitude})"
```

Comment:
I think for this it might be better to explicitly set the value we are expecting in the incidentType03 

e.g 

`val lat = arbitrary[String].sample.value`
`val long = arbitrary[String].sample.value`
`val incidentType = incidentType03.copy(Coordinates = Coordinates(lat, long)`
then doing something like 

`result.value.value mustBe s"(102.156, -109.423)"` or `result.value.value mustBe s"(lang, long)"` if using arbitrary values when explicitly setting the value within the incident type 03

Will help to avoid any flaky test if the Coordinates are not generated as part of the arbitrary incident type 03